tests/transformationTest.py

```diff
@@ -1,15 +1,22 @@
 from src.transformation import Transformation
 import unittest
 import os
+import shutil
 import csv
 
 
 class TransformationTest(unittest.TestCase):
-    def setUp(self):
-        if os.getenv('KBC_DATADIR') is not None:
-            self.data_dir = os.getenv('KBC_DATADIR')
-        else:
-            self.data_dir = '.'
+    @classmethod
+    def setUpClass(self):
+        # Container is in the production runned with "www-data" user,
+        # so we want to test it in the same way.
+        # Code directory is not writtable for "www-data" user, so we use /tmp dir.
+        tests_dir = '/code/tests'
+        tmp_dir = '/tmp/tests'
+        if os.path.exists(tmp_dir) and os.path.isdir(tmp_dir):
+            shutil.rmtree(tmp_dir)
+        shutil.copytree(tests_dir, tmp_dir)
+        self.data_dir = tmp_dir
```

Comment:
Ak spustam testy pod `www-data` uzivatelom, tak nie som schopny zapisovat do `/code/tests/...`, ... a testy su tak napisane, ze niektore subory generuju. Preto sa cely obsah testov skopiruje do `/tmp/tests` a tam sa to da upravovat. Je to lepsi aj preto, lebo sa ti zbytocne nezapisu docasne subory do repozitara.

Reply:
Namiesto `setUp` sa pouziva `setUpClass`, ... nemozeme pred kazdym testom zmazat temp adresar, ... pretoze Python ma uz v pamati nacitane Python subory z tohto adresara a pada to ... teda nasetupuje sa to pred prvym testom.